src/Repositories/LocalSettingsRepository.php

```diff
@@ -71,6 +71,17 @@ public function getPath(SearchIndex $index): string
 
         $name = is_array($name) ? current($name) : $name;
 
+        if (config('scout.algolia.settings_path')) {
+            if (! $this->files->exists(config('scout.algolia.settings_path'))) {
+                $this->files->makeDirectory(config('scout.algolia.settings_path'), 0755, true);
+            }
+
+            return implode(DIRECTORY_SEPARATOR, [
+                config('scout.algolia.settings_path'),
+                'scout-'.Str::lower($name).'.php',
```

Comment:
`'scout-'.Str::lower($name).'.php'` this piece of code can be reused on the line 85 declaring one variable on the line 73.

Reply:
Moved path to a variable for re-use as suggested.